salalib/linkutils.cpp

```diff
@@ -95,6 +95,68 @@ namespace depthmapX {
                       [&](const PixelRefPair &pair)->void{ currentMap.mergePixels(pair.a,pair.b); });
     }
 
+    void unmergePixelPairs(const std::vector<PixelRefPair>& links, PointMap& currentMap) {
```

Comment:
Where does this code come from? Unlinking was available from the GUI, so I guess you copied it?
Can we make sure we use the same copy of the code in both places?

Reply:
Unlinking from the GUI is available but the process differs because the points are first selected through a different part of the GUI that only picks up filled cells. For the CLI we need to test for this before unlinking. The GUI also makes the assumption that unlinking should be done by clicking on one of the cells (I suspect for speed) while here both ends of the link are requested. Thus we can also test whether the unlink requested actually exists as a link and pop an error (for correctness).